docs/advisors/checks/mongodb-xfs.md

```diff
@@ -0,0 +1,26 @@
+# MongoDB XFS Filesystem type
+
+## Description
+This check returns a warning if DBPATH is not using the XFS filesystem type.
+
+## Resolution
+
+As per [MongoDB Documentation](https://www.mongodb.com/docs/v6.0/administration/production-notes/#std-label-prod-notes-linux-file-system), when using a WiredTiger storage engine, use of the XFS file system type is strongly recommended for data bearing nodes to avoid performance issues that may occur when using EXT4 with the WiredTiger engine.
```

Comment:
```suggestion
To avoid performance issues when using EXT4 with WiredTiger, [MongoDB Documentation](https://www.mongodb.com/docs/v6.0/administration/production-notes/#std-label-prod-notes-linux-file-system) strongly recommends using the XFS file system type with WiredTiger storage engine for data bearing nodes. 
```